src/Http/Http.Extensions/src/HeaderDictionaryTypeExtensions.cs

```diff
@@ -204,7 +204,7 @@ internal static IList<T> GetList<T>(this IHeaderDictionary headers, string name)
 
             if (StringValues.IsNullOrEmpty(values))
             {
-                return null;
+                new List<T>();
```

Comment:
Can use Array.Empty instead of `new List` to not allocate
```suggestion
                Array.Empty<T>();
```

Reply:
@benaadams I think I Should change it back to `new List`, because you get a exception when calling `Add` on `Array.Empty`. What's your thought?